Makefile

```diff
@@ -246,7 +246,7 @@ husky:
 LICENSE_EYE = $(shell pwd)/bin/license-eye
 .PHONY: license-eye
 license-eye: ## Download license-eye locally if necessary. https://github.com/apache/skywalking-eyes
-	$(call go-get-tool,$(LICENSE_EYE),github.com/apache/skywalking-eyes/cmd/license-eye@v0.3.0)
+	$(call go-get-tool,$(LICENSE_EYE),github.com/apache/skywalking-eyes/cmd/license-eye@main)
```

Comment:
>always prefer to use a revision instead of `main`.

According to [official suggestion](https://github.com/apache/skywalking-eyes#check-license-headers)

Reply:
OK. I will change it to the latest version.